package/src/monaco.d.ts

```diff
@@ -21,8 +21,16 @@ declare module monaco.languages.kusto {
         disabledCompletionItems?: string[];
         onDidProvideCompletionItems?: monaco.languages.kusto.OnDidProvideCompletionItems;
         enableHover?: boolean;
+        formatter?: FormatterOptions;
     }
 
+    export interface FormatterOptions {
```

Comment:
duplicate with settings.ts, probably forgot to remove it + LanguageSettings

Reply:
no, it is not auto-generated. This file exposes anything that monaco-kusto exposes. There is a comment on it that explains the motivation which I didn't get into too much.